### PR TITLE
Improved displaying in nebula console

### LINKED ISSUE #825: when result is empty, head information should not be displayed
when result is empty, head information should not be displayed 

case:
```
(1)  (user@127.0.0.1) [myspace]> show tags;
========
| Name |
========
Got 0 rows (Time spent: 978/1633 us)


(2)  show edges;
========
| Name |
========
Got 0 rows (Time spent: 1845/2788 us)

(3)
(user@127.0.0.1) [myspace]> create tag  player(name string, age int);
Execution succeeded (Time spent: 3024/4232 us)

(user@127.0.0.1) [myspace]> show tags;
==========
| Name   |
==========
| player |
----------
Got 1 rows (Time spent: 2356/3419 us)

(user@127.0.0.1) [myspace]> fetch prop on player 1004;
============================
| player.name | player.age |
============================
Execution succeeded (Time spent: 2918/4050 us)
```

### PR DESCRIPTION
**Pull request by moneytoyang**
fix #825

### CODE REVIEW

File: src/console/CmdProcessor.cpp [hunk 1]
```diff
@@ -336,12 +336,16 @@ void CmdProcessor::processServerCmd(folly::StringPiece cmd) {
...
         }
-        printResult(resp);
-        if (resp.get_rows() != nullptr) {
+        if (resp.get_rows() && !resp.get_rows()->empty()) {
+            printResult(resp);
             std::cout << "Got " << resp.get_rows()->size()
                       << " rows (Time spent: "
                       << resp.get_latency_in_us() << "/"
                       << dur.elapsedInUSec() << " us)\n";
+        } else if (resp.get_rows() && resp.get_rows()->empty()) {
```
Comment (dutor) [inline src/console/CmdProcessor.cpp]: `resp.get_rows()->empty()` here is always true if `resp.get_rows() != nullptr`

Comment (moneytoyang) [reply src/console/CmdProcessor.cpp]: true?   false ?

Comment (moneytoyang) [reply src/console/CmdProcessor.cpp]: The result displays as follow:
```
(user@127.0.0.1) [myspace]> show tags;
==========
| Name   |
==========
| player |
----------
Got 1 rows (Time spent: 1185/1733 us)

(user@127.0.0.1) [myspace]> show edges;
Empty set (Time spent: 1740/2362 us)
```

I had used gdb  to debug as follows: 

In nebula::graph::CmdProcessor::processServerCmd  function, use show edges as above:

```
(gdb) p	resp.get_rows()
$1 = (std::vector<nebula::graph::cpp2::RowValue, std::allocator<nebula::graph::cpp2::RowValue> > *) 0x7ffe68672250
(gdb) p	resp.get_rows()->empty()
$2 = true
```

Comment (dutor) [reply src/console/CmdProcessor.cpp]: I meant the test `(resp.get_rows() && resp.get_rows()->empty())` is identical to `(resp.get_rows() != nullptr)`

Comment (moneytoyang) [reply src/console/CmdProcessor.cpp]: I don't think so.
 case:
```
(user@127.0.0.1) [myspace]> show tags;
==========
| Name   |
==========
| player |
----------
Got 1 rows (Time spent: 1185/1733 us)

(user@127.0.0.1) [myspace]> show edges;
Empty set (Time spent: 1740/2362 us)
```

when `show tags`,  execute the branch `(resp.get_rows() && !resp.get_rows()->empty())`
when `show edges`,  execute the branch `(resp.get_rows() && resp.get_rows()->empty())`

Comment (dutor) [reply src/console/CmdProcessor.cpp]: I think you did not quite understand me.

Comment (moneytoyang) [reply src/console/CmdProcessor.cpp]: when `resp.get_rows() != nullptr `,   the value of` resp.get_rows()->empty()` is true or false.
In above case, execute` show tags`, resp.get_rows()->empty()  is false
execute` show edges`, `resp.get_rows()->empty()`  is true

Comment (dutor) [reply src/console/CmdProcessor.cpp]: Upon reaching the `else if ()` branch, the `if ()` branch was not been chosen, right? 

Thus `resp.get_rows() == nullptr` or `resp.get_rows().empty() == true`, right?

Thus in the `else if ()` branch, if `resp.get_rows() != nullptr`, then `resp.get_rows().empty() == true`, right?

Comment (moneytoyang) [reply src/console/CmdProcessor.cpp]: Yeah.

**Comment by dutor** [comment]
Close due to unresponsive

**Comment by moneytoyang** [comment]
please refer  #825 

(1)  (user@127.0.0.1) [myspace]> show tags;
========
| Name |
========
Got 0 rows (Time spent: 978/1633 us)


(2)  show edges;
========
| Name |
========
Got 0 rows (Time spent: 1845/2788 us)

(3)
(user@127.0.0.1) [myspace]> fetch prop on player 1004;
============================
| player.name | player.age |
============================
Execution succeeded (Time spent: 2918/4050 us)


**I don't think the header information should be returned when the execution result is empty.
This design feels too bad.**

**Comment by dutor** [comment]
@moneytoyang Please address the comments above, thanks.

**Comment by moneytoyang** [comment]
The result after the fix is as follows：
image

**Comment by dutor** [comment]
I mean to update this PR and  change `        } else if (resp.get_rows() && resp.get_rows()->empty()) {` to `        } else if (resp.get_rows()) {`

**Comment by moneytoyang** [comment]
> I mean to update this PR and change ` } else if (resp.get_rows() && resp.get_rows()->empty()) {` to ` } else if (resp.get_rows()) {`

Yeah, I wil update it, thx

**Comment by dutor** [comment]
Jenkins, please run tests.

**Pull request merged**